models/action.go

```diff
@@ -382,7 +382,7 @@ func activityQueryCondition(opts GetFeedsOptions) (builder.Cond, error) {
 	}
 
 	if opts.Date != "" {
-		dateLow, err := time.Parse("2006-01-02", opts.Date)
+		dateLow, err := time.ParseInLocation("2006-01-02", opts.Date, time.Local)
```

Comment:
We have a location setting on app.ini, I think we should use that.

Reply:
Yes, you are right.

Reply:
I have pushed the modification. But I have another question, is it necessary to pass the browser time zone or time zone offset to backend (not only this scene, like github.com passes a cookie called tz)?